release-notes.txt

```diff
@@ -1,3 +1,541 @@
+* at86rf2xx: add missing handler for TRX24_RX_START interrupt
+* periph_rtt: add periph_rtt_set_counter  feature
+
+
+Documentation
+-------------
+
++ riotboot and its USB DFU version were added to the documentation
++ cpu/cc26xx_cc13xx: add CPU documentation
+* doc/SAUL: State expectations on blocking and interrupts
+* sys/phydat: clarify use of phydat::scale in phydat_fit
+* sys/ztimer: clarify doc on clock ops
+* doc/Coding_Convention: clarify: void return is allowed
+* sys/event: add missing precondition to doc
+
+Build System / Tooling
+----------------------
+
+* RIOT now defaults to C11
+* RIOT now compiles with `-fwrapv`
++ compile_commands.json files can be produced, easing IDE integration
+* riotboot USB DFU: Enter bootloader when board's bootloader button is pressed on power-up
+* Make: enable SECONDEXPANSION globally
+
+Testing
+-------
++ Test utilities result output (turo) module for testing logic abstraction
++ Github integration of static tests using annotations completed
+
+Kconfig
+-------
++ Modules with dependency model added:
+    + drivers/{l, m, n, o, p, q, r, s, t, u, v, w}*
+    + pkg/c25519
+    + pkg/cayenne-lpp
+    + pkg/cifra
+    + pkg/cn-cbor
+    + pkg/driver_atwinc15x0
+    + pkg/driver_bme680
+    + pkg/emlearn
+    + pkg/gemmlowp
+    + pkg/hacl
+    + pkg/heatshrink
+    + pkg/jsmn
+    + pkg/libb2
+    + pkg/libcose
+    + pkg/libfixmath
+    + pkg/libhydrogen
+    + pkg/lora-serialization
+    + pkg/micro-ecc
+    + pkg/microcoap
+    + pkg/minmea
+    + pkg/monocypher
+    + pkg/nanocbor
+    + pkg/nanopb
+    + pkg/qDSA
+    + pkg/qcbor
+    + pkg/relic
+    + pkg/talking_leds
+    + pkg/tiny-asn1
+    + pkg/tinycbor
+    + pkg/tinycrypt
+    + pkg/tlsf
+    + pkg/tweetnacl
+    + pkg/u8g2
+    + pkg/ubasic
+    + pkg/ucglib
+    + pkg/umorse
+    + pkg/utensor
+    + pkg/yxml
+    + sys/{crypto, hashes, random}
+    + sys/analog_util
+    + sys/app_metadata
+    + sys/base64
+    + sys/bitfield
+    + sys/bloom
+    + sys/cb_mux
+    + sys/eepreg
+    + sys/matstat
+    + sys/memarray
+    + sys/mineplex
+    + sys/oneway
+    + sys/posix_{sleep, inet}
+    + sys/sema
+    + sys/seq
+    + sys/uri_parser
+    + sys/timex
+    + sys/ztimer
+
++ Modules with configurations exposed:
+    + cpu/stm32f2f4f7: clock configuration
+    + net/lorawan
+    + sys/shell
+
++ Source EXTERNAL_MODULES configurations
+
+Examples
+--------
+* examples/suit_update: update to aiocoap 0.4.1
+* examples/dtls-sock: add multiple credentials
+* examples/gnrc_border_router: enable ethernet uplink
+
+API Changes
+-----------
+
+* mtd: rename mtd_write_page() -> mtd_write_page_raw(), add high-level mtd_write_page()
+* sys/ztimer: rename required_pm_mode to block_pm_mode
+* sock/async: add function to retrieve session object of current DTLS event
+* sys/net/netopt: make NETOPT_TX_END_IRQ and friends read-only
+
+Deprecations
+============
+
+Removals
+--------
+- Remove deprecated features:
+  - core/lifo
```

Comment:
This corresponds to #16101 and is not a complete removal but a deprecation (as described in the PR).

Reply:
ah, I misread, will move it again

Reply:
fixed